fanuc_r2000ib_support/package.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>fanuc_r2000ib_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support package for the Fanuc R-2000iB (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Fanuc R-2000iB manipulators. This currently includes the /210F
+      variant only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>R-2000iB/210F - "J1 - Normal Range"</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information in
+      the <em>FANUC R-2000iB Mechanical Unit Operator's Manual</em>
+      version <em>B-82234EN/11</em>. All urdfs are based on the default motion
```

Comment:
I've added a specific ref to the documentation which provided information on the joint limits et al., as that will make it easier to verify/correct them in the future.